### PR TITLE
docs(topics): add projects.md document

### PR DESCRIPTION
**Pull request by technosophos**
Document configuring projects.

### CODE REVIEW

File: docs/topics/install.md [hunk 1]
```diff
@@ -40,15 +43,17 @@ $ helm install brigade/brigade --set image.name=my-image --set image.tag=1.2.3
 There are a variety of other configuration options for Brigade. Run `helm fetch values ./chart/brigade`
 to see them all.
 
-### Disabling RBAC
+### Enabling RBAC (optional)
 
-By default, Brigade has Role Based Access Control support. To disable this, set
-`rbac.enabled` to `false`:
+By default, Brigade has Role Based Access Control suppor turned offt. To enable this, set
```
Comment (bacongobbler) [inline docs/topics/install.md]: off*

Comment (technosophos) [reply docs/topics/install.md]: Fixed

File: docs/topics/install.md [hunk 2]
```diff
@@ -40,15 +43,17 @@ $ helm install brigade/brigade --set image.name=my-image --set image.tag=1.2.3
 There are a variety of other configuration options for Brigade. Run `helm fetch values ./chart/brigade`
 to see them all.
 
-### Disabling RBAC
+### Enabling RBAC (optional)
 
-By default, Brigade has Role Based Access Control support. To disable this, set
-`rbac.enabled` to `false`:
+By default, Brigade has Role Based Access Control suppor turned off. To enable this, set
```
Comment (bacongobbler) [inline docs/topics/install.md]: support*

Comment (technosophos) [reply docs/topics/install.md]: Fixed

**Pull request merged**